--- a/fbpanel/.config/help
+++ b/fbpanel/.config/help
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
 import re, os, sys, textwrap
 # Formats help message
--- a/fbpanel/.config/repl.py
+++ b/fbpanel/.config/repl.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
 import re, sys
 
--- a/fbpanel/.config/tar.py
+++ b/fbpanel/.config/tar.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
 import subprocess as sp
 import re, tempfile
--- a/fbpanel/configure
+++ b/fbpanel/configure
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
 import sys
 if sys.version_info < (2, 7):
--- a/fbpanel/panel/panel.c
+++ b/fbpanel/panel/panel.c
@@ -936,6 +936,9 @@
     exit(1);
 }
 
+void
+configure(xconf *xc);
+
 int
 main(int argc, char *argv[])
 {
--- a/fbpanel/plugins/battery/os_linux.c
+++ b/fbpanel/plugins/battery/os_linux.c
@@ -119,6 +119,9 @@
     RET(ret);
 }
 
+power_supply*
+power_supply_parse(power_supply* ps);
+
 static gboolean
 battery_update_os_sys(battery_priv *c)
 {
