diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index cafab16..35fb7c6 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -5,5 +5,4 @@
 pkgbase=evince
 pkgname=(
   evince
-  evince-lib-docs
 )
diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index cafab16..86485c1 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -48,3 +48,4 @@ pkgver() {
 prepare() {
   cd evince
+  git apply --verbose ~/.config/makepkg/evince-src.patch
 }
