diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index cafab16..35fb7c6 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -5,5 +5,4 @@
 pkgbase=evince
 pkgname=(
   evince
-  evince-lib-docs
 )
diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index cafab16..86485c1 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -48,3 +48,16 @@ pkgver() {
 prepare() {
   cd evince
+  git apply --verbose << EOF
+diff --git a/shell/ev-window.c b/shell/ev-window.c
+index 4d8629cd..9be0d596 100644
+@@ -1476,5 +1476,8 @@ setup_document_from_metadata (EvWindow *window)
+ 			request_width = MIN (request_width, monitor_width);
+ 			request_height = MIN (request_height, monitor_height);
+ 		}
++		// limit initial window size to 2/3 of monitor size
++		request_width = monitor_width*2/3;
++		request_height = monitor_height*2/3;
+ 
+ 		if (request_width > 0 && request_height > 0) {
+EOF
 }
