--- fbpanel/.config/help
+++ fbpanel/.config/help
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
 import re, os, sys, textwrap
 # Formats help message

--- fbpanel/.config/repl.py
+++ fbpanel/.config/repl.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
 import re, sys
 

--- fbpanel/.config/tar.py
+++ fbpanel/.config/tar.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
 import subprocess as sp
 import re, tempfile

--- fbpanel/configure
+++ fbpanel/configure
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
 import sys
 if sys.version_info < (2, 7):

--- fbpanel/panel/panel.c
+++ fbpanel/panel/panel.c
@@ -936,6 +936,9 @@
     exit(1);
 }
 
+void
+configure(xconf *xc);
+
 int
 main(int argc, char *argv[])
 {

--- fbpanel/plugins/battery/os_linux.c
+++ fbpanel/plugins/battery/os_linux.c
@@ -119,6 +119,9 @@
     RET(ret);
 }
 
+power_supply*
+power_supply_parse(power_supply* ps);
+
 static gboolean
 battery_update_os_sys(battery_priv *c)
 {

--- fbpanel/plugins/deskno/deskno.c
+++ fbpanel/plugins/deskno/deskno.c
@@ -53,7 +53,7 @@
 static gint
 name_update(GtkWidget *widget, deskno_priv *dc)
 {
-    char buffer [15];
+    char buffer [20];
 
     ENTER;
     dc->deskno = get_net_current_desktop();
