--- a/.config/help
+++ b/.config/help
@@ -1,2 +1,2 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
--- a/.config/repl.py
+++ b/.config/repl.py
@@ -1,2 +1,2 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
--- a/.config/tar.py
+++ b/.config/tar.py
@@ -1,2 +1,2 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
--- a/configure
+++ b/configure
@@ -1,2 +1,2 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 
--- a/panel/panel.c
+++ b/panel/panel.c
@@ -938,4 +938,7 @@
 
+void
+configure(xconf *xc);
+
 int
 main(int argc, char *argv[])
 {
--- a/plugins/battery/os_linux.c
+++ b/plugins/battery/os_linux.c
@@ -121,4 +121,7 @@
 
+power_supply*
+power_supply_parse(power_supply* ps);
+
 static gboolean
 battery_update_os_sys(battery_priv *c)
 {
--- a/plugins/deskno/deskno.c
+++ b/plugins/deskno/deskno.c
@@ -53,5 +53,5 @@
 static gint
 name_update(GtkWidget *widget, deskno_priv *dc)
 {
-    char buffer [15];
+    char buffer [20];
 
--- a/.config/rules.mk
+++ b/.config/rules.mk
@@ -157,7 +157,7 @@ error: 'g_type_class_add_private' is deprecated
 ifeq ($(origin CFLAGS),undefined)
 ifeq ($(origin DEBUG),undefined)
-CFLAGS = -O2 -Wall -Werror
+CFLAGS = -O2 -Wall
 else
-CFLAGS = -O0 -Wall -Werror
+CFLAGS = -O0 -Wall
 endif
 endif
